### PR TITLE
rearrange

### PR DESCRIPTION
**Pull request by Ashxsh1**
Rearranged the Home About and Services section.

### CODE REVIEW

**Comment by Susmita-Dey** [comment]
@Ashxsh1 Issue no?

**Comment by Ashxsh1** [comment]
Issue no 54, i am doing it in bits.

**Comment by Susmita-Dey** [comment]
@Ashxsh1 Your branch has merge conflicts. Fix them.

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Comment by Susmita-Dey** [comment]
@Ashxsh1 Any updates?

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

**Comment by myhacktech** [comment]
@Ashxsh1 updates please...

File: index.html [hunk 1]
```diff
@@ -265,6 +207,66 @@ <h2 >Consult A Doctor</h2>
...
+          <img src="/images/shayan.jpeg" alt="shayan" loading="lazy">
+          <h5>Shayan Pal</h5>
+          <div class="pra">
+            <p>Member</p>
+            <a href="https://bio.link/shayanpal"><button>Let's Connect</button></a>
+          </div>
+        </div>
+
+        <div class="card">
+          <img src="/images/soumyadeep.jpeg" alt="soumyadeep"loading="lazy">
```
Comment (AliReza1083) [inline index.html:229]: There should be a space between each attribute.
For writing clean and nice, it is better to give a space between them.
```diff
- alt="soumyadeep"loading="lazy"
+ alt="soumyadeep" loading="lazy"
```

File: index.html [hunk 2]
```diff
@@ -265,6 +207,66 @@ <h2 >Consult A Doctor</h2>
...
+            <a href="https://bio.link/chaitri"><button>Let's Connect</button></a>
+          </div>
+        </div>
+
+        <div class="card">
+          <img src="/images/sasanka.jpeg" alt="sasanka"loading="lazy">
+          <h5>Sasanka Kundu</h5>
+          <div class="pra">
+            <p>Member</p>
+            <a href="https://bio.link/sasankakundu"><button>Let's Connect</button></a>
```
Comment (AliReza1083) [inline index.html:260]: If you are using a tag, There is no need use button inside of it.
You can design it to look like a button.
```diff
- <a href=""><button></button></a>
+ <a href=""></a>
```
> **Note**
> For styling a tag, If you want to add `padding` on it, set the display to equal to inline-block.

> **Warning**
> If you don't use inline-block, The `padding-top` and `padding-bottom` is not going to work.

**Review by AliReza1083 — COMMENTED** [review]
I reviewed your codes, and I came up with this.
Wish you all the best.

**Comment by Ashxsh1** [comment]
My laptop was damaged due to which I couldn't work for the last week...I'll
get to it now.

**Comment by Susmita-Dey** [comment]
@Ashxsh1 Closing this issue. You can make a new PR

**Pull request closed without merging**